APPLY_ACCESS_TOKEN.md

```diff
@@ -0,0 +1,29 @@
+# Apply for a nodejs-github-bot token
+
+Automation in https://github.com/nodejs may require github access token to access
+permission scoped endpoints. In the case of such requirement, the access token can
+be requested to be created under the name of https://github.com/nodejs-github-bot. 
+
+Only fine-grained access token can be created under the name of https://github.com/nodejs-github-bot.
+
+To create a fine-grained access token with nodejs-github-bot, submit a PR to
+add the requested repo in the registry below, and a TSC member or a build WG member
+(who has access to the https://github.com/nodejs-github-bot account) needs to take
+action to create the fine-grained token and save the token to the requested repo's
```

Comment:
We should document the precise process for how to land such PR, create the secret, add it as a repo secret.